packages/renderer/index.d.ts

```diff
@@ -215,7 +210,7 @@ declare namespace ReactPDF {
     children: string;
   }
 
-  class Note extends React.Component<React.PropsWithChildren<NoteProps>> {}
+  class Note extends React.Component<NoteProps> {}
```

Comment:
While the current type does limit `children` to a single `string`, the error message is convoluted.
> No overload matches this call.
  Overload 1 of 2, '(props: PropsWithChildren\<NoteProps> | Readonly\<PropsWithChildren\<NoteProps>>): Note', gave the following error.
    Type 'Element' is not assignable to type 'string & ReactNode'.
      Type 'ReactElement\<any, any>' is not assignable to type 'string & ReactPortal'.
        Type 'ReactElement\<any, any>' is not assignable to type 'string'.
  Overload 2 of 2, '(props: PropsWithChildren\<NoteProps>, context: any): Note', gave the following error.
    Type 'Element' is not assignable to type 'string & ReactNode'.

vs the new error
> No overload matches this call.
  Overload 1 of 2, '(props: NoteProps | Readonly\<NoteProps>): Note', gave the following error.
    Type 'Element' is not assignable to type 'string'.
  Overload 2 of 2, '(props: NoteProps, context: any): Note', gave the following error.
    Type 'Element' is not assignable to type 'string'.